### PR TITLE
Mi-go Camp Starting Scenario

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
```SUMMARY: Content "Mi-go Camp Starting Scenario"```

#### Purpose of change
Adds a new challenge start: the Mi-go Camp. You can start either as a naked captive or as a somewhat geared up would be rescuer. This is a winter start, since the mi-go only start capturing people later after the cataclysm.

#### Describe the solution
Everything was already here, just made the two profession and the scenario

#### Testing
Start game
Choose the scenario
It starts, no crash, no error

#### Additional context
I'm open to balance suggestion for the rescuer gear, and point cost of this
This scenario might no be winnable

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -774,6 +774,40 @@
       "female": [ "panties", "bra", "stockings", "dress_shoes", "maid_dress", "maid_hat" ]
     }
   },
+  {
+    "type": "profession",
+    "ident": "captive",
+    "name": { "male": "Captive", "female": "Captive" },
+    "description": "You were folowing a road at night trying to get away from the horrors of the city when you heard a voice calling out in the dark.  As you stepped away to investigate you suddenly felt a seering pain in your head and blacked out.  You just woke up in this place... Are you even on earth anymore?",
+    "points": 0,
```
Comment (anothersimulacrum) [inline data/json/professions.json]: I think starting with absolutely nothing should probably earn you points.

**Comment by thethunderhawk** [comment]
I would be surprised if this were winnable. I think if it turns out to be pretty much unfeasible, it’d just be bloat on the menu and maybe a trap for newer players.

**Comment by Night-Pryanik** [comment]
Interesting... I like it.

File: data/json/professions.json (hunk 1 above)
Comment (ZhilkinSerg) [reply data/json/professions.json]: 1 point like Shower Victim (or maybe 2) should be fine.

File: data/json/professions.json [hunk 2]
```diff
@@ -774,6 +774,40 @@
...
+    "flags": [ "SCEN_ONLY" ]
+  },
+  {
+    "type": "profession",
+    "ident": "rescuer",
+    "name": { "male": "Rescuer", "female": "Rescuer" },
+    "description": "You were ready, you went in heads on determined to find and rescue your friends.  But now as you walk through those strange corridors the atmosphere grows heavy on you and you're not so sure anymore, you might be the one in need of a rescue now.",
```
Comment (KorGgenT) [inline data/json/professions.json]:
```suggestion
    "description": "You were ready.  You went in determined to find and rescue your friends.  But now as you walk through those strange corridors, the atmosphere grows heavyand you're not so sure anymore. You might be the one in need of a rescue now.",
```
Reads a little better this way, but i'm not super clear what you meant by "you went in heads on determined" so it's possible that could be reworded properly.

File: data/json/scenarios.json [hunk 3]
```diff
@@ -393,6 +393,17 @@
     ],
     "flags": [ "LONE_START", "CHALLENGE", "OUTDOOR" ]
   },
+  {
+    "type": "scenario",
+    "ident": "migo_prisoner",
+    "name": "Challenge - Mi-Go Camp",
+    "points": -8,
+    "description": "You find yourself in the most alien place you've ever seen, the hot humid air and the organic aspect of the structure makes you feel like you're trapped inside a giant creature.  Whatever it is, you need to get out of here before They find you.",
```
Comment (KorGgenT) [inline data/json/scenarios.json]:
```suggestion
    "description": "You find yourself in the most alien place you've ever seen.  the hot humid air and the organic aspect of the structure makes you feel like you're trapped inside a giant creature.  Whatever it is, you need to get out of here before They find you.",
```
A sentence break in here makes sense.

File: data/json/professions.json [hunk 4]
```diff
@@ -774,6 +774,40 @@
       "female": [ "panties", "bra", "stockings", "dress_shoes", "maid_dress", "maid_hat" ]
     }
   },
+  {
+    "type": "profession",
+    "ident": "captive",
+    "name": { "male": "Captive", "female": "Captive" },
+    "description": "You were folowing a road at night trying to get away from the horrors of the city when you heard a voice calling out in the dark.  As you stepped away to investigate you suddenly felt a seering pain in your head and blacked out.  You just woke up in this place... Are you even on earth anymore?",
```
Comment (KorGgenT) [inline data/json/professions.json]:
```suggestion
    "description": "You were following a road at night trying to get away from the horrors of the city, when you heard a voice calling out in the dark.  As you stepped away to investigate, you suddenly felt a searing pain in your head and blacked out.  You just woke up in this place... Are you even on earth anymore?",
```

**Comment by Amoebka** [comment]
Sounds cool, but it pretty much boils down to "walk around - see a migo - start running - exhaust stamina - die in less than 50 turns total".

**Comment by ZhilkinSerg** [comment]
> 
> 
> Sounds cool, but it pretty much boils down to "walk around - see a migo - start running - exhaust stamina - die in less than 50 turns total".

Challenge starts are meant to be harsh.

**Pull request merged**